migtests/tests/pg-tests/pg-codependent-schemas/validate

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import yb
+
+def main():
+	yb.run_checks(migration_completed_checks)
+
+
+#=============================================================================
+
+EXPECTED_ROW_COUNT_PUBLIC = {
+	'audit2': 4,              
+	'foreign_test': 2,
+ 	'list_active': 2,  
+ 	'list_archived': 1,  
+ 	'list_part': 5,   
+	'primary_test': 3,
+	'tt':4,
+	'v1':2,
+	'mv1':2,
+}
+EXPECTED_ROW_COUNT_SCHEMA2 = {
+	'audit': 4,
+	'foreign_test': 3,
+	'list_others': 2, 
+	'primary_test': 2,
+	'v1': 2,
+}
+
+def check_validations_per_schema(tgt,schema_name):
+
+	EXPECTED_ROW_COUNT = EXPECTED_ROW_COUNT_PUBLIC if schema_name == 'public' else EXPECTED_ROW_COUNT_SCHEMA2
+
+	table_list = tgt.get_table_names(schema_name)
+	print("table_list:", table_list)
+	assert (len(table_list) == 7 and schema_name == 'public') or (len(table_list) == 4 and schema_name == 'schema2')
+
+	view_list = tgt.get_objects_of_type("VIEW", schema_name)
+	print("view_list:", view_list)
+	assert len(view_list) == 1
+
+	mview_list = tgt.get_objects_of_type("MVIEW", schema_name)
+	print("mview_list:", mview_list)
+	assert (len(mview_list) == 1 and schema_name == 'public') or (len(mview_list) == 0 and schema_name == 'schema2')
+
+	for table_name, row_count in EXPECTED_ROW_COUNT.items():
+		cnt = tgt.get_row_count(table_name,schema_name)
+		print(f"table_name: {schema_name}.{table_name}, row_count: {cnt}")
+		assert row_count == cnt
```

Comment:
You can compare two full dictionaries to have exactly the same set of key-value pairs simply using `==` operator. No need to compare each element individually.

You can rename `cnt` with a more pronounciable `count`.

Space after comma, always: you can go through the PEP8 once (https://peps.python.org/pep-0008/). You can also consider running pylint on your scripts to find such errors.